src/pharmpy/tools/structsearch/tool.py

```diff
@@ -99,31 +99,38 @@ def run_tmdd(context, model):
 
 
 def run_pkpd(context, model):
-    wf = create_fit_workflow(model)
+    pk_dataset = model.dataset
+    pk_dataset = pk_dataset[pk_dataset["DVID"] != 2]
+    pk_model = model.replace(
+        dataset=pk_dataset,
+        description=model.description + ". Removed rows with DVID=2.",
+        name="PK_" + model.name,
+    )
```

Comment:
Personal preference but I think to make it a bit more unified maybe there should be a function in `.pkpd` called `create_pk_model`?

Reply:
Added a function but it might have to be removed later since we do not want to use a copy of the input model.